### ISSUE TITLE
HtmlFormatter - Responsive html chord

### ISSUE THREAD

**Issue by artutra** [martijnversluis/ChordSheetJS#34]
How about an option to allow the htmlFormatter to render chord pro
from this:
```
Some r[C]andom lyrics
```
to this:
```html
<p>Some r<div class="chord">C</div>andom lyrics</p>
```
So it would be possible to use css to make a responsive design like this
![cresponsivity2](https://user-images.githubusercontent.com/15269475/32730930-3dcee57c-c867-11e7-8285-d9a091557f31.gif)
ps: Ignore the tablature

**Comment by martijnversluis** [comment]
Nice suggestion @artutra! I can think of two possible solutions:

- the option to make the tags configurable, such as passing `{ row_tag: 'tr', chord_tag: 'th', lyrics_tag: 'td' }` to the constructor of `HtmlFormatter`
- the option to choose between several layouts (table layout, responsive layout)

What do you think would be the best option?

**Comment by artutra** [comment]
I believe that the second option will fit better on this case, which does not exclude the possibility to configure the tags or classes too.

**Comment by martijnversluis** [comment]
@artutra I just released version `2.0.0` which includes the `HtmlDivFormatter`. Please refer to the `README` for usage.
